SA/project_documentation/content/project_method.typ

```diff
@@ -0,0 +1,6 @@
+= Project Method
+Using Scrum for this project isn't an option for us, since that would create a massive amount of meetings that doesn't make sense for a project with just 2 people.
```

Comment:
I think both versions are correct.
The `doesn't` is correct, since it references `a massive amount`, but `don't` is correct too since it references `meetings`.
So it depends on what we want to say. Do we not want to have the meetings, or just less meetings?

Reply:
Ah, yes that make sense. 

I don't really care, so we can leave it as it